docs/src/Typeahead.doc.js

```diff
@@ -353,4 +360,65 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="zIndex"
+    name="With ZIndex"
+    description={`
+    Modals within Layer components with zIndexs will overlay any children component. Use zIndex prop in Typeahead to bring the Typeahead list to the top of the zIndex stack.`}
```

Comment:
I think more verbosity could communicate a bit better here. Perhaps something like
"When Typeahead is used within a parent component that has a z-index set, a z-index will also need to be set on the Typeahead. Otherwise the Typeahead will render behind the parent component in the stacking context."

Reply:
+1 ^

Reply:
DONE. Thanks, much betta.